src/safe_apps/admin.py

```diff
@@ -2,5 +2,36 @@
 
 from .models import SafeApp, Provider
 
-models = [SafeApp, Provider]
-admin.site.register(models)
+
+class NetworksFilter(admin.SimpleListFilter):
+    title = 'Networks'
+    parameter_name = 'networks'
+
+    def lookups(self, request, model_admin):
+        values = SafeApp.objects.values_list('networks', flat=True)
+        # lookups requires a tuple to be returned – (value, verbose value)
+        networks = [(network, network) for networks in values for network in networks]
+        networks = sorted(set(networks))
+        print(networks)
+        return networks
+
+    def queryset(self, request, queryset):
+        value = self.value()
+        if value:
+            queryset = queryset.filter(networks__contains=[value])
+        return queryset
+
+
+@admin.register(SafeApp)
+class SafeAppAdmin(admin.ModelAdmin):
+    list_display = ('name', 'url', 'networks')
+    list_filter = (NetworksFilter,)
```

Comment:
Setting here `networks` does not create the list_filter for you?

Reply:
The list was created but unfortunately only with the different sets:

<img width="300" src="https://user-images.githubusercontent.com/3332770/116710967-a4eaa700-a9d2-11eb-9883-78c9cd0f6a31.png" />

By flattening the list with `NetworksFilter` we can have the following set:

<img width="300" src="https://user-images.githubusercontent.com/3332770/116711056-c0ee4880-a9d2-11eb-8f50-e633f977e5df.png" />

Reply:
Nice, thanks, just curious!